### DISCUSSION TITLE
Looking to contribute to landscape2

### DISCUSSION THREAD

**Discussion by pennantt** [cncf/landscape2#697] [Q&A]
Hi, I want to contribute to the landscape2 project and was wondering if anyone is currently working something that may be relevant for junior developers. My background lends itself to developing with TypeScript so I would be interested in what I could contribute to at that level.

**Comment by tegioz** [comment]
Hi @thomasx-0 👋

Thank you for your interest in the landscape2 project!

At the moment I don't think we have anything that matches that criteria. But when an interesting good first issue that we need help with comes up, we'll add the labels `help wanted` and `good first issue` to it 🙂

In the meantime, I would recommend you to take a look at <https://clotributor.dev>. It lists +1.5k issues from Cloud Native projects that are looking for help (all issues on it are labelled as `help wanted`!), and you can use the filters to find the ones that best suit your interests and skillset.
